wp-parsely.php

```diff
@@ -312,7 +328,7 @@ public function insert_parsely_page() {
         // TODO: Maping of an install's post types to Parse.ly post types (namely page/post)
         $parselyPage = array(
             "@context" => "http://schema.org",
-            "@type" => "WebPage"
+            "@type" => "NewsArticle"
```

Comment:
This should stay WebPage as the default- it gets overridden in 357 if the post is in fact a post, and if this is WebPage that means that non-post pages will get NewsArticle as their @type too.